dev/io.openliberty.microprofile.telemetry.1.0.internal/src/io/openliberty/microprofile/telemetry/internal/cdi/OpenTelemetryProducer.java

```diff
@@ -28,8 +28,8 @@
 public class OpenTelemetryProducer {
 
     private static final String INSTRUMENTATION_NAME = "io.openliberty.microprofile.telemetry";
-    private static final String ENV_ENABLE_PROPERTY = "OTEL_EXPERIMENTAL_SDK_ENABLED";
-    private static final String CONFIG_ENABLE_PROPERTY = "otel.experimental.sdk.enabled";
+    private static final String ENV_ENABLE_PROPERTY = "OTEL_SDK_DISABLED";
+    private static final String CONFIG_ENABLE_PROPERTY = "otel.sdk.disabled";
```

Comment:
The constant names should change to match the meaning of the property. I.e. `ENV_ENABLE_PROPERTY`.